lighthouse-core/audits/manual/pwa-cross-browser.js

```diff
@@ -7,6 +7,17 @@
 'use strict';
 
 const ManualAudit = require('./manual-audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that suggests to the user that a site should work across different browsers. */
```

Comment:
probably should follow the positive template here even though there's no failureTitle (since it's a manual audit)